app/ide-desktop/lib/content-config/src/config.json

```
@@ -123,6 +123,11 @@
         "newDashboard": {
           "value": false,
           "description": "Determines whether the new dashboard with cloud integration is enabled."
+        },
+        "profiling": {
+          "value": false,
+          "description": "Enable the option to profile workflows.",
```

I wouldn't use the word "workflows" here--we have *profiling workflows* that are automated sequences of events for profiling non-interactively, and the profile button is not related to that

---

Alright. Any ideas on more accurate terminology?

---

@kazcw I think that our terminology should be changed, unfortunately. From the user perspective, what they are building in Enso is a workflow. So I believe it is correct to say that we will have "profile workflow button" in IDE in the future.

Regarding this description though, I find it not clear. What do you mean by "enable the option" - what is "option"?

---

Isn't most of what's defined here options? Things the user can enable / disable / set to a certain value? Like line 106 in that file says: "Options that enable experimental features that are not yet stable." Happy to change it if you find some other wording clearer.

---

```suggestion
          "description": "Enable the button to profile workflows.",
```

Let's make it as literal as it can be. And please, let's spend less time on trivial changes like this one. If anybody has objections – suggest your own wording.